tests/metrics/utils/prompts/test_v1.py

```diff
@@ -0,0 +1,184 @@
+import re
+
+from mlflow.metrics.base import EvaluationExample
+from mlflow.metrics.utils.prompts.v1 import EvaluationModel
+
+
+def test_evaluation_model_output():
+    model1 = EvaluationModel(
+        name="correctness",
+        definition="Correctness refers to how well the generated output matches "
+        "or aligns with the reference or ground truth text that is considered "
+        "accurate and appropriate for the given input. The ground truth serves as "
+        "a benchmark against which the provided output is compared to determine the "
+        "level of accuracy and fidelity.",
+        grading_prompt="""Correctness: If the answer correctly answer the question, below are the
+        details for different scores:
+        - Score 0: the answer is completely incorrect, doesn’t mention anything about the question
+        or is completely contrary to the correct answer.
+        - Score 1: the answer provides some relevance to the question and answer one aspect
+        of the question correctly.
+        - Score 2: the answer mostly answer the question but is missing or hallucinating on one
+        critical aspect.
+        - Score 4: the answer correctly answer the question and not missing any major aspect
+        """,
+        examples=[
+            EvaluationExample(
+                input="This is an input",
+                output="This is an output",
+                score=4,
+                justification="This is a justification",
+                variables={"ground_truth": "This is an output"},
+            )
+        ],
```

Comment:
can we add a test case with multiple examples?

Reply:
Sure. I can add that